Tests/scripts/collect_tests/collect_tests.py

```diff
@@ -927,6 +932,22 @@ def _collect_yml(self, content_item_path: Path) -> CollectionResult | None:
                 allow_incompatible_marketplace=override_support_level_compatibility,
             )
 
+    def _collect_integrations_using_apimodule(self, api_module_id) -> CollectionResult | None:
+        integrations_using_apimodule = self.id_set.api_modules_to_integrations.get(api_module_id, [])
+        collection_result_of_apimodule = None
+        for integration in integrations_using_apimodule:
+            try:
+                integration_collected = self._collect_yml(integration.path)
+                if collection_result_of_apimodule:
+                    collection_result_of_apimodule += integration_collected
+                else:
+                    collection_result_of_apimodule = integration_collected
+            except Exception as e:
+                logger.info(e)
```

Comment:
add to the log - why do we skip in this case, rather than crash everything? If we tolerate it, devs may not notice it